test/integration/page/PageWPDE.spec.ts

```diff
@@ -36,6 +31,19 @@ describe( 'PageWPDE', function () {
 		expect( retrievedCampaignParameters.startDate ).toBe( '2023-11-01' );
 	} );
 
+	it( 'returns campaign parameters for thankyou banners', () => {
+		window.thankYouCampaignParameters = {
+			progressBarPercentage: 80,
+			numberOfDonors: 42,
+			numberOfMembers: 23
+		};
+		const page = new PageWPDE( tracking );
+
+		const retrievedCampaignParameters = page.getThankYouCampaignParameters();
+
+		expect( retrievedCampaignParameters.numberOfDonors ).toBe( 42 );
```

Comment:
Instead of doing a spot check you could do
```suggestion
		expect( retrievedCampaignParameters ).toEqual( window.thankYouCampaignParameters );
```